pkg/utils/pod.go

```diff
@@ -213,3 +215,23 @@ func PodToleratesTaints(pod *v1.Pod, taintsOfNodes map[string][]v1.Taint) bool {
 	}
 	return false
 }
+
+// GetPodOwnerRefKey returns pod's ownerReference key
+func GetPodOwnerRefKey(pod *v1.Pod) []string {
+	podOwnerRefKey := []string{}
+	ownerRefList := pod.ObjectMeta.GetOwnerReferences()
+	if len(ownerRefList) == 0 {
+		klog.Errorf("pod does not have any ownerRefs")
+	}
+	imageList := []string{}
+	for _, container := range pod.Spec.Containers {
+		imageList = append(imageList, container.Image)
+	}
+	sort.Strings(imageList)
+	imagesHash := strings.Join(imageList, "#")
+	for _, ownerRef := range ownerRefList {
```

Comment:
It is sufficient to only take `ownerRefList` into account. The imageHash was needed to improve detection of what a duplicate is. Not needed here.